### DISCUSSION TITLE
Oracdc does not captured changed data

### DISCUSSION THREAD

**Discussion by iamjavadev1** [averemee-si/oracdc#49] [Q&A] [answered]
Hi Averemee-si,
            
          I've tried debugging code and found that the oracdc process must wait for an online redo log to be copied to an archived redo log file? It then needs to wait for a switch redo log file to occur before it can send changed data to Kafka. Am I correct?

Regards,
Iamjavadev1

**Comment by averemee-si** [comment, marked as answer]
Hi iamjavadev1,

**oracdc** connector reads changed data from archived logs. For more information about archived redo log filws please refer to Oracle Database documentation. A good starting point would be [Archived Redo Log Files](https://docs.oracle.com/en/database/oracle/oracle-database/23/cncpt/physical-storage-structures.html#GUID-4E3F6CAA-453F-4518-83B9-B3A3CCF11B2E) which contains
... ... ...
You can use archived redo log files to:
- Recover a database backup
- Update a [standby database](https://docs.oracle.com/en/database/oracle/oracle-database/23/cncpt/glossary.html#GUID-728D4956-0F56-4D39-A63A-2E3FF5CCEDE4)
- Obtain information about the history of a database using the Oracle LogMiner utility
... ... ...
For more information on how to reduce the lag between archived and online redo logs , see [Near real time: is it possible?](https://github.com/averemee-si/oracdc/wiki/Near-real-time:-is-it-possible%3F)

Regards,
Aleksei
